server/server.go

```diff
@@ -122,6 +123,40 @@ func (server *HTTPServer) listOfClustersForOrganization(writer http.ResponseWrit
 	}
 }
 
+func getTotalRuleCount(reportRules types.ReportRules) int {
```

Comment:
strictly speaking it should be `uint`, but people don't like unsigned numbers :)